### PR TITLE
Add documentation for the La Marzocco calendar

### PR DESCRIPTION
**Pull request by zweckj**
## Proposed change

Add documentation for the La Marzocco calendar
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/108237
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/lamarzocco.markdown [hunk 1]
```diff
@@ -83,3 +83,7 @@ Host:
 | Prebrew/-infusion mode | Whether to use prebrew, preinfusion, or neither | Disabled, Prebrew, Preinfusion | Linea Micra, Linea Mini, GS3 AV |
 | Steam level | The level your steam boiler should run at | 1,2,3 | Linea Micra |
 
+## Calendar
+
+The integration exposes a calendar which represents the auto on/off schedule set for the machine. The schedule will be displayed recurringly, that means if you set the machine to be turn on on Mondays at 8:00, and off at 9:00, you will get events for all Mondays in your calendar. On days where you have the auto on/off feature disabled, you won't get an event in the calendar. Also, if you have the auto on/off feature disabled globally (e.g. through the switch "Auto on/off"), there will be no events in the calendar.
```
Comment (c0ffeeca7) [inline source/_integrations/lamarzocco.markdown]:
```suggestion
The integration exposes a calendar for the auto on/off schedule set for the machine. The schedule will be displayed recurringly: If you set the machine to start up on Mondays at 8:00, and shut down at 9:00, you will get events for all Mondays in your calendar. On days when you have the auto on/off feature disabled, you won't get an event in the calendar. Also, if you have the auto on/off feature disabled globally (for example, through the switch "Auto on/off"), there will be no events in the calendar.
```

**Review by c0ffeeca7 — COMMENTED** [review]
small tweaks

**Review by c0ffeeca7 — DISMISSED** [review]
Thank you, @zweckj :+1:

**Comment by zweckj** [comment]
parent merged

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @zweckj :+1:

**Pull request merged**